### PR TITLE
Adding 3 mini apps into a new folder structure for companies/investments

### PR DESCRIPTION
**Pull request by paulgain**
new-folder-structure

https://uktrade.atlassian.net/browse/IPBETA-320

### CODE REVIEW

File: src/apps/companies/apps/investments/growth-capital-profile/views/list.njk [hunk 1]
```diff
@@ -1,5 +1,6 @@
-{% extends "../template.njk" %}
+{% extends "../../../../views/template.njk" %}
 
 {% block body_main_content %}
+  <pre>{{ localNavItems | dump(2) }}</pre>
```
Comment (yeahfro) [inline src/apps/companies/apps/investments/growth-capital-profile/views/list.njk]: Do you want this?

File: src/apps/companies/apps/investments/large-capital-profile/views/list.njk [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% extends "../template.njk" %}
+{% extends "../../../../views/template.njk" %}
 
 {% block body_main_content %}
 <h2 class="govuk-heading-m">Investments - Large capital profile</h2>
```
Comment (yeahfro) [inline src/apps/companies/apps/investments/large-capital-profile/views/list.njk]: Missing space

File: src/apps/companies/apps/investments/growth-capital-profile/views/list-deprecated.njk [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{% extends "../template.njk" %}
+{% extends "../../../../views/_deprecated/template.njk" %}
```
Comment (yozzo) [inline src/apps/companies/apps/investments/growth-capital-profile/views/list-deprecated.njk]: should we be using `deprecated` templates?

File: src/apps/companies/apps/investments/growth-capital-profile/views/list.njk (hunk 1 above)
Comment (yozzo) [inline src/apps/companies/apps/investments/growth-capital-profile/views/list.njk]: is this a logger?

File: src/apps/companies/apps/investments/growth-capital-profile/views/list-deprecated.njk (hunk 3 above)
Comment (paulgain) [reply src/apps/companies/apps/investments/growth-capital-profile/views/list-deprecated.njk]: I'm following a pattern throughout Data Hub. It relates to the vertical/horizontal navigation. Once we are using the horizontal navigation in its entirety the deprecated views will be pulled - that's my take on it. Is this correct @yeahfro?

File: src/apps/companies/apps/investments/growth-capital-profile/views/list.njk (hunk 1 above)
Comment (paulgain) [reply src/apps/companies/apps/investments/growth-capital-profile/views/list.njk]: Ha, good spot. I'll remove this.

File: src/apps/companies/apps/investments/large-capital-profile/views/list.njk (hunk 2 above)
Comment (paulgain) [reply src/apps/companies/apps/investments/large-capital-profile/views/list.njk]: Yep.

File: src/apps/companies/apps/investments/growth-capital-profile/views/list.njk (hunk 1 above)
Comment (paulgain) [reply src/apps/companies/apps/investments/growth-capital-profile/views/list.njk]: I was debugging. I'll remove it no problem. It's not really a problem as there is nothing in this view and it's hidden from the user in any case.

**Pull request merged**